### ISSUE TITLE
WCAG 2.1: 1.4.13 tooltip in modal dialog is not dismissible

### ISSUE THREAD

**Issue by ferBonnin** [microsoft/accessibility-insights-web#777]
## Describe the bug
related to S.C. 1.4.13, tooltip in modal dialog is not dismissable using keyboard only.

**To Reproduce**
1. Go to any page and continue an assessment
2. Dialog to continue or start new assessment appear
3. using keyboard, navigate through the links
4. try to press esc to dismiss tooltip, you can't

## Expected behavior
tooltip should be dismissable using esc key

## Screenshots

![Screenshot tooltip in the modal dialog](https://user-images.githubusercontent.com/9062104/59379432-6a5ac100-8d0b-11e9-8301-2b7abf414dab.png)

## Extension (please complete the following information)

- OS: [e.g. Windows/Mac] Windows 
- Version [e.g. 1.140.1] 2.4.1
- Browser Version [e.g Chrome 71.0] Version 74.0.3729.169
- Target Page [e.g www.bing.com] any

## Are you willing to submit a PR?

If asked, will you be willing to submit a PR to fix this bug? No

## Did you search for similar existing issues?

Yes

**Comment by dbjorge** [comment]
Office fabric issue; you can see the same behavior in their default tooltip example at https://developer.microsoft.com/en-us/fabric#/controls/web/tooltip

Filed https://github.com/OfficeDev/office-ui-fabric-react/issues/9435

**Comment by dbjorge** [comment]
Office fabric issue has been fixed, just need to wait for a new fabric release now.

**Comment by ferBonnin** [comment]
this bug will be part of the Office Fabric update feature

**Comment by ferBonnin** [comment]
@smoralesd this is fixed, however, now the tab stop doesn't stop in the second link

**Comment by dbjorge** [comment]
Verified both original issue and new tab ordering issue are fixed in master
